src/panels/config/integrations/integration-panels/zha/dialog-zha-reconfigure-device.ts

```diff
@@ -225,10 +226,11 @@ class DialogZHAReconfigureDevice extends LitElement {
                   )}
                 </h2>
 
-                ${this._clusterConfigurationStatuses!.size > 0
+                ${this._clusterConfigurationStatuses &&
+                this._clusterConfigurationStatuses.size > 0
```

Comment:
```suggestion
                ${this._clusterConfigurationStatuses?.size
```

Reply:
I had that first... something is up with my dev environment. It still flags that with the "object possibly null or undefined" when I use `?.`

Reply:
actually it breaks CI too...

Reply:
never mind... needed more coffee